src/components/step-flow/step-flow.spec.tsx

```diff
@@ -358,4 +423,87 @@ describe("Step Flow component", () => {
       });
     }
   );
+
+  describe("when ref handle is passed to StepFlow", () => {
+    it("calling exposed focus method refocuses on StepFlow's root container", async () => {
+      const MockComponent = () => {
+        const stepFlowHandle = React.useRef<StepFlowHandle>(null);
+
+        return (
+          <div>
+            <StepFlow
+              totalSteps={5}
+              currentStep={1}
+              ref={stepFlowHandle}
+              title="foo"
+            />
+            <Button onClick={() => stepFlowHandle.current?.focus()}>
+              Press me to refocus on Dialog
+            </Button>
+          </div>
+        );
+      };
+
+      const user = userEvent.setup();
+      const { container } = render(<MockComponent />);
+      const button = screen.getByRole("button");
+      button.focus();
```

Comment:
comment: I don't think you need to focus the button here (just click) and given the what this is testing it may be a little confusing